site/content/extend/plugins/webapp/reference.md

```diff
@@ -96,3 +111,80 @@ The theme object has the following properties:
 | mentionHighlightLink | Color of text for mention links in posts |
 | codeTheme | Code block theme, either 'github', 'monokai', 'solarized-dark' or 'solarized-light' |
 
+## Exported Libraries and Functions
```

Comment:
Do you mean `Exposed` libraries?

Reply:
Nah I wanted `Exported` but I do tend to use it interchangeably with exposed